packages/card/src/Card.js

```diff
@@ -43,6 +53,22 @@ const Root = createStyledComponent(
 /**
  * Card component
  */
-export default function Card(props: Props) {
-  return <Root {...props} />;
+export default class Card extends Component {
```

Comment:
Why convert this to a class based component?  You could probably just create a standalone `onKeyPress` function.  If for some reason it must be a class, you should be able to extend `PureComponent` instead of `Component`.